dev-ml/dose3/dose3-7.0.0-r1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
```

Comment:
Have you tried to do DUNE_PKG_NAME=$PN here

and add 
src_compile() {
    dune-compile $PN
}

maybe you can remove the src_prepare

}

Reply:
that worked, thanks.